app/controllers/financial_transactions_controller.rb

```diff
@@ -23,41 +23,33 @@ def new
     @financial_transaction.transactable_id = params[:transactable_id] if @financial_transaction.transactable_type != Payment.name
   end
 
-  # GET /financial_transactions/1/edit
-  def edit; end
-
-  # POST /financial_transactions
   def create
     @financial_transaction = FinancialTransaction.new(financial_transaction_params)
     if @financial_transaction.transactable_type == Payment.name
-      payment = Payment.new(payment_params) # hard fail
+      payment = Payment.new(payment_params)
       unless payment.save
         flash[:danger] = 'Please Properly Fill Out Contact And Payment Fields'
```

Comment:
Would it be better to be clearer here, like so:
`flash[:danger] = payment.errors.full_messages`?
That is, if your application.html.erb is set up to accept flash[:danger] as an array.

Reply:
If that's outside of the scope of your PR, never mind--I just think it would be good to do.

Reply:
I agree, but I chose not to address that here because it will be addressed when I eventually finish #382